dynawo/sources/Modeler/DataInterface/PowSyblIIDM/test/TestCriteria.cpp

```diff
@@ -358,6 +358,68 @@ createBusBreakerNetworkWithGenerators(double busV, double busVNom, double pow1,
   return network;
 }
 
+static boost::shared_ptr<powsybl::iidm::Network>
+createNetworkWithCustomisableNumberOfLoadsAndGenerators(double busV, double busVNom, const std::vector<int>& loadPowers) {
+  auto network = boost::make_shared<powsybl::iidm::Network>("MyNetwork", "MyNetwork");
+
+  auto substationAdder = network->newSubstation()
+                                     .setId("MySubStation")
+                                     .setName("MySubStation_NAME")
+                                     .setTso("TSO");
+
+  substationAdder.setCountry(powsybl::iidm::Country::FR);
+  powsybl::iidm::Substation& substation = substationAdder.add();
+
+  powsybl::iidm::VoltageLevel& vl1 = substation.newVoltageLevel()
+                                     .setId("MyVoltageLevel")
```

Comment:
Please make sure that we do not test several times the same load/generator (by checking failingCriteria_ for example)